bin/log-failed-release

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
```

Comment:
Could you add a comment on what this command does, or add a usage instruction here?
```
echo "what it does"
echo "Usage: $0 KUBE_CONTEXT [NAMESPACE] RELEASE_NAME [ERROR]"
echo "KUBE_CONTEXT  ..."
echo "NAMESPACE  ... (default: default)"
...
```